### PR TITLE
Image & Deploy Improvements

### PR DESCRIPTION
**Pull request by ianwestcott**
Asana Tasks:

- [Fix API Checker deployment](https://app.asana.com/0/881264583703207/1199906233124798/f)
- [update api_checker to use hexpm/elixir container](https://app.asana.com/0/1113179098808463/1199910231398959/f)

## Considerations

- Use `hexpm/elixir` base image instead of `elixir`
- Update to Elixir 1.11 and Alpine 13.3
- Resolve deployment errors by removing additional parameters from the exported task definition
- Increase wait time for ECS deployment from 60 to 120 seconds

### CODE REVIEW

**Comment by ianwestcott** [comment]
I should note that this method of updating the task definition is relatively bespoke and unorthodox, but I'm hoping to standardize it when I import this application into Terraform, so I opted against any major revisions in the meantime.

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM elixir:1.10-alpine as builder
+FROM hexpm/elixir:1.11.0-erlang-21.3.8.21-alpine-3.13.1 as builder
```
Comment (paulswartz) [inline Dockerfile:1]: Is there a .tool-versions file? If so, it should also be updated with these versions.

Comment (ianwestcott) [reply Dockerfile:1]: There is, and awkwardly, it specifies a newer Erlang version than what's provided by this base image. I'm going to try a different image&mdash;there are new builds as of this morning&mdash;and see about using the most recent versions of everything.

Comment (ianwestcott) [reply Dockerfile]: See https://github.com/mbta/api_checker/pull/23/commits/bdcb880013e4254a20132aa179fc9023139fad7f. I bumped Erlang to 23.x, hopefully that won't be an issue but let me know if I should keep it on 22.

Comment (ianwestcott) [reply Dockerfile]: Test failures imply that maybe Erlang 23 is too ambitious. https://semaphoreci.com/mbta/api_checker/branches/idw-image-deploy/builds/3 I'll see about backing out to 22 for now.

Comment (paulswartz) [reply Dockerfile]: Weird, that failure doesn't seem to make any sense for Erlang 23 🤷

Comment (ianwestcott) [reply Dockerfile]: Got another failure on https://github.com/mbta/api_checker/pull/23/commits/e6c6c2e006b4a6f5a83983b6ba46d9f908dd269b. I'm officially out of my depth here. Any recommendations?

Comment (paulswartz) [reply Dockerfile]: Looks like it's on `main` too: https://semaphoreci.com/mbta/api_checker/branches/main/builds/2

This is likely caused by the dates we use for holiday checking falling out of the API: I'll update the code with some new dates.

**Comment by ianwestcott** [comment]
Closing in favor of #24

**Pull request closed without merging**